src/xdocs/config_blocks.xml

```diff
@@ -363,41 +363,41 @@ switch (a)
         </p>
         <source>
 &lt;module name=&quot;EmptyCatchBlock&quot;&gt;
-    &lt;property name=&quot;commentFormat&quot; value=&quot;This is expected&quot;/&gt;
-    &lt;property name=&quot;exceptionVariableName&quot; value=&quot;myException&quot;/&gt;
+  &lt;property name=&quot;commentFormat&quot; value=&quot;This is expected&quot;/&gt;
+  &lt;property name=&quot;exceptionVariableName&quot; value=&quot;myException&quot;/&gt;
 &lt;/module&gt;
         </source>
         <p>
           Such empty blocks would be suppressed:
         </p>
         <source>
 try {
-    throw new RuntimeException();
+  throw new RuntimeException();
 } catch (RuntimeException e) {
     //This is expected
```

Comment:
Only 2 spaces indent.

Reply:
@rnveach Sorry, I didn't understand your comment clearly. The indentation after try block is 2 (After my changes). What should I change in that?

Reply:
@varunu28 My comment is on the comment inside the `catch` block. It is indented 4 spaces over from the catch, which is where the throw was originally.
So move `//This is expected` left 2 spaces.

Reply:
@rnveach Done